tests/schemas/api_v3/generatorSchema.schema.js

```diff
@@ -30,7 +30,7 @@ const generatorSchema = {
 	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
 	name: Joi.string().pattern(regex.NAME).optional(),
 	publicKey: Joi.string().pattern(regex.PUBLIC_KEY).allow(null).optional(),
-	nextAllocatedTime: Joi.number().integer().min(getCurrentTimestamp()).required(),
+	nextAllocatedTime: Joi.number().integer().min(getCurrentTimestamp() - 20).required(),
```

Comment:
Please add a TODO to investigate this later. Or, better add it to the acceptance criteria in https://github.com/LiskHQ/lisk-service/issues/1491.